src/applications/personalization/dashboard/mocks/lighthouse-claims/index.js

```diff
@@ -0,0 +1,73 @@
+const addDays = require('date-fns/addDays');
+const format = require('date-fns/format');
+
+const createLighthouseClaimsSuccess = (updatedDaysAgo = 1, open = true) => {
+  const daysAgo = addDays(new Date(), -updatedDaysAgo);
```

Comment:
you can use subDays here instead and it might be a little less confusing

https://date-fns.org/v2.30.0/docs/subDays